common/src/main/scala/explore/graphql/todo/ToggleMutation.scala

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2016-2020 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore.graphql
+
+import clue.GraphQLQuery
+import explore.model.Task
+import io.circe.{ Decoder, Encoder }
+import io.circe.generic.semiauto.{ deriveDecoder, deriveEncoder }
+
+object ToggleMutation extends GraphQLQuery {
+  val document = """
+      mutation ToggleMutation($taskId: String!) {
+        toggle(id: $taskId) {
+          id
+          title
+          completed
+        }
+      }"""
+
+  case class Variables(taskId: String)
```

Comment:
Yes, we could do a number of things here.

For the moment, the classes in `explore.graphql` have been auto-generated using https://github.com/muuki88/sbt-graphql, and then slightly modifying the resulting code manually to fit our needs. If we decide to use this approach, we could very easily modify the generator so that it generates the code we need.

However, for simple queries I think it might make sense to declare GraphQL statements directly in the code, without the need of a boilerplate object. I'll explore this approach shortly.